### PR TITLE
Premium template upgrades/fixes: Stripe + discount code closes#533

### PR DESCRIPTION
**Pull request by vinayakveil**
closes https://github.com/suredone/automations/issues/533

Stripe upgrade and discount code HTML clean up for the following companies
-  carpartstemplate
-  shoestemplate
-  innoinco

### CODE REVIEW

**Comment by vinayakveil** [comment]
@jasonspalace 

As per https://github.com/suredone/automations/issues/533 issue's reference:

- Added stripe by taking default-2019 as reference 
- Changed HTML content for discount view.

File: carpartstemplate/form-cardcheckout.htm [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {# card processing form #}
 {% set wpsets = sd_option('site_cart_card_processor')=='worldpay' and sd_option('worldpay_settings') ? json_decode(sd_option('worldpay_settings'),true) : [] %}
 <div class="sd-cc-wrapper">
+{% if not stripe %}
```
Comment (jasonspalace) [inline carpartstemplate/form-cardcheckout.htm:4]: @velsvelmurugan where is `stripe` set?

File: innoinco/form-cardcheckout.htm [hunk 2]
```diff
@@ -1,10 +1,11 @@
 {# card processing form #}
 {% set wpsets = sd_option('site_cart_card_processor')=='worldpay' and sd_option('worldpay_settings') ? json_decode(sd_option('worldpay_settings'),true) : [] %}
 <div class="sd-cc-wrapper">
+{% if not stripe %}
```
Comment (jasonspalace) [inline innoinco/form-cardcheckout.htm:4]: @velsvelmurugan where is `stripe` set?

File: shoestemplate/form-cardcheckout.htm [hunk 3]
```diff
@@ -1,10 +1,11 @@
 {# card processing form #}
 {% set wpsets = sd_option('site_cart_card_processor')=='worldpay' and sd_option('worldpay_settings') ? json_decode(sd_option('worldpay_settings'),true) : [] %}
 <div class="sd-cc-wrapper">
+{% if not stripe %}
```
Comment (jasonspalace) [inline shoestemplate/form-cardcheckout.htm:4]: @velsvelmurugan where is `stripe` set?

**Review by jasonspalace — CHANGES_REQUESTED** [review]
@velsvelmurugan was this tested? please see notes.

**Comment by vinayakveil** [comment]
@jasonspalace 
RE1:
No, it's not tested. I requested a review so I can test once you are ok with the changes.
RE2:
I just changed code in reference to https://github.com/suredone/templates/blob/master/default-2019/form-cardcheckout.htm#L103-L189 
Do I need to do like we did for other templates eg: https://github.com/suredone/automations/pull/391#issuecomment-769808022
?

**Comment by jasonspalace** [comment]
@velsvelmurugan ok looks like it needs more work:

- the reference to `stripe` is being passed in from [`form-checkoutsteps.htm`](https://github.com/suredone/suredone/blob/master/flavors/default-2019/form-checkoutsteps.htm#L153) 
- also ensure you set the variable in this file
- must be applied to all carpartstemplate, innoinco, shoestemplate

**Comment by jasonspalace** [comment]
@velsvelmurugan here is some reference to make it easier..

Screen Shot 2021-08-16 at 12 52 19 PM
Screen Shot 2021-08-16 at 12 52 54 PM


Internal reference https://github.com/suredone/suredone/pull/2489

**Review by jasonspalace — APPROVED** [review]

**Comment by vinayakveil** [comment]
@jasonspalace 

- Uploaded template files and assets of carpartstemplate, shoestemplate, innoinco in our ror test account.
- Tested Stripe Action

### Attachments:

### carpartstemplate:

orderplaced
stripecarparts
addressfillcarparts
cartforcarparts


-----


### shoestemplate
stripeforshoestemplate
shoestemplateaddrform
showtemplcarts

orderplacedshoestemplate


-----

### innoinco

incoorderplaced
incoaddressform
incocart
incostripe

**Pull request merged**